src/utils/devLogger.ts

```diff
@@ -13,42 +13,45 @@ export type DevLoggerPrefix =
   | "THEME_VALUES_TO_APPLY";
 
 export class DevLogger {
-  constructor(private enabled: boolean = false) {
-    if (!this.enabled) {
+  private static enabled: boolean;
+
+  constructor(enabled: boolean = false) {
```

Comment:
Any reason for the changes to DevLogger? I prefer how it was.

Reply:
Nvm, saw your comment about making it static.

Reply:
enabled only gets set once, via YSS passing down the query params. So either this needs to be static or the devLogger instance needs to be passed down as a prop from editor anywhere we want to use it.